python/semantic_kernel/semantic_functions/prompt_template_config.py

```diff
@@ -15,65 +15,67 @@ class PromptTemplateConfig(SKBaseModel, Generic[AIRequestSettingsT]):
     schema_: int = Field(default=1, alias="schema")
     type: str = "completion"
     description: str = ""
-    completion: AIRequestSettingsT = Field(default_factory=AIRequestSettings)
+    execution_settings: AIRequestSettingsT = Field(default_factory=AIRequestSettings)  # todo: this should be a dict
```

Comment:
Let's make a note to also rename the request settings to ExecutionSettings?

Reply:
Good idea, will add a work item. Thanks!